api_spec.yaml

```diff
@@ -576,7 +576,8 @@ paths:
         - OAuth2 Recipe
       operationId: addOAuth2ScopeToClient
```

Comment:
Semantic error at paths./appid-<appId>/recipe/oauth2/client/scope.delete.requestBody

Reply:
not able to reproduce in swagger.

Reply:
this is till not solved

Reply:
issue resolved.